.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Versions**
+    - Vaadin / Flow version:
```

Comment:
It's a corner-case to have a Flow version different from what's coming from the platform, and in most cases Flow version is not relevant.

CE version on the other hand will not be coming from the platform (for now), and is also more likely relevant for the bug.
```suggestion
    - Vaadin version:
    - Collaboration Engine version:
```